### PR TITLE
GCW-2691 fix minor issues in corporate donor

### PR DESCRIPTION
**Pull request by abhinav098**
Minor fixes:
- remove validation from crm-id
- Add cancel button on new donor screen
- Display correct mobile number on donor_details screen
- Increase the size of icons on donor_details page for small screens so that it easy to click through mobile as well

### CODE REVIEW

File: app/templates/companies/new.hbs [hunk 1]
```diff
@@ -30,17 +30,14 @@
...
-      {{t 'donor_details.company.crm_id'}}
+      {{t 'donor_details.company.crm_id'}} {{t 'donor_details.company.optional'}}
     </div>
   </div>
 
   <div class="row">
     <div class="{{if isMobileApp 'small-7 large-8' 'small-12'}} columns">
       {{#form-control}}
-        {{numeric-input value=crmId id="crm-id" name="crm-id" maxlength="21" required=true class='company-input' pattern="\d{1,8}"}}
-        <div class='input-error'>
```
Comment (Darshan-Chauhan) [inline app/templates/companies/new.hbs]: Any specific reason for removing validation message?

Comment (abhinav098) [reply app/templates/companies/new.hbs]: Matt asked to remove it.

Comment (abhinav098) [reply app/templates/companies/new.hbs]: @Darshan-Chauhan

Comment (Darshan-Chauhan) [reply app/templates/companies/new.hbs]: Ok, can you also remove the translation for validation message?

Comment (abhinav098) [reply app/templates/companies/new.hbs]: and its a numeric field, so user wont be able to enter string characters. So removed it.

Comment (abhinav098) [reply app/templates/companies/new.hbs]: > Ok, can you also remove the translation for validation message?

Sure, Actually i used the same translation for placeholder, so i'll change the key name.
Thanks

Comment (abhinav098) [reply app/templates/companies/new.hbs]: Done 👍

File: app/templates/companies/new.hbs [hunk 2]
```diff
@@ -20,27 +20,24 @@
...
-          {{t 'donor_details.company.crm_id_validation'}}
-        </div>
+        {{numeric-input value=crmId id="crm-id" name="crm-id" maxlength="21" class='company-input' pattern="\d{1,8}" placeholder=(t 'donor_details.company.crm_id_validation')}}
```
Comment (Darshan-Chauhan) [inline app/templates/companies/new.hbs]: I think you'll need to change placeholder here :)

Comment (abhinav098) [reply app/templates/companies/new.hbs]: ohh yes, sorry my bad.

**Pull request merged**